app/components/Views/TrendingView/components/SectionCard/SectionCard.tsx

```diff
@@ -12,17 +15,46 @@ const createStyles = (theme: Theme) =>
       paddingVertical: 16,
       paddingHorizontal: 16,
       backgroundColor: theme.colors.background.muted,
-      borderColor: theme.colors.border.muted,
+      borderWidth: 0,
     },
   });
+interface SectionCardProps {
+  sectionId: SectionId;
+}
 
-const SectionCard: React.FC<PropsWithChildren> = ({ children }) => {
+const SectionCard: React.FC<SectionCardProps> = ({ sectionId }) => {
+  const navigation = useNavigation();
   const theme = useAppThemeFromContext();
   const styles = useMemo(() => createStyles(theme), [theme]);
 
+  const { data, isLoading } = SECTIONS_CONFIG[sectionId].useSectionData();
+
+  const renderFlatItem: ListRenderItem<unknown> = useCallback(
+    ({ item }) => {
+      const section = SECTIONS_CONFIG[sectionId];
+      return section.renderRowItem(item, navigation);
+    },
+    [navigation, sectionId],
+  );
+
   return (
     <Card style={styles.cardContainer} disabled>
-      {children}
+      {isLoading && (
+        <>
+          {SECTIONS_CONFIG[sectionId].renderSkeleton()}
+          {SECTIONS_CONFIG[sectionId].renderSkeleton()}
+          {SECTIONS_CONFIG[sectionId].renderSkeleton()}
```

Comment:
nit, is is possible to make these compound components instead of render functions?
```suggestion
          <SECTIONS_CONFIG[sectionId].Skeleton />
          <SECTIONS_CONFIG[sectionId].Skeleton />
          <SECTIONS_CONFIG[sectionId].Skeleton />
```